apps/web/src/components/borrow/table/AvailableToBorrowTable.tsx

```diff
@@ -1,55 +1,83 @@
-import React, { FunctionComponent } from 'react'
-import { Button, CellProps, Table } from '@bloxifi/ui'
+import React, { useState, FunctionComponent } from 'react'
+import { Button, ColumnData, Table } from '@bloxifi/ui'
+import { useTranslation } from 'react-i18next'
 
 import { AssetName } from '../AssetName'
 import { FormattedNumber } from '../FormattedNumber'
+import { BorrowModal } from '../modal/BorrowModal'
 
-import { WalletContainer } from '@/containers/WalletContainer'
+import { ReservesData, WalletContainer } from '@/containers/WalletContainer'
 
 export const AvailableToBorrowTable: FunctionComponent = () => {
+  const { t } = useTranslation()
   const {
-    state: { reserves },
+    state: {
+      reserves,
+      userAccountData: { healthFactor },
```

Comment:
@aleksaToljic It won't be updated automatically, we need to refetch the data that we get from contract (wallet balance & userAccountData for now). 
I haven't done this yet because I was not sure which data should be refetched after borrow/deposit/collateral.

I suggest to create a new Jira issue for refetching data on borrow, deposit & collateral, because It will be something similar for each transaction. What do you think?